src/components/Buttons.js

```diff
@@ -1,48 +1,42 @@
-          <button type="button" onClick={this.handleButtonClick}>%</button>
-          <button type="button" onClick={this.handleButtonClick}>7</button>
-          <button type="button" onClick={this.handleButtonClick}>8</button>
-          <button type="button" onClick={this.handleButtonClick}>9</button>
-          <button type="button" onClick={this.handleButtonClick}>4</button>
-          <button type="button" onClick={this.handleButtonClick}>5</button>
-          <button type="button" onClick={this.handleButtonClick}>6</button>
-          <button type="button" onClick={this.handleButtonClick}>1</button>
-          <button type="button" onClick={this.handleButtonClick}>2</button>
-          <button type="button" onClick={this.handleButtonClick}>3</button>
-          <button id="zero" type="button" onClick={this.handleButtonClick}>0</button>
-          <button type="button" onClick={this.handleButtonClick}>.</button>
-        </div>
-        <div id="right">
-          <button type="button" onClick={this.handleButtonClick}>÷</button>
-          <button type="button" onClick={this.handleButtonClick}>x</button>
-          <button type="button" onClick={this.handleButtonClick}>-</button>
-          <button type="button" onClick={this.handleButtonClick}>+</button>
-          <button type="button" onClick={this.handleButtonClick}>=</button>
-        </div>
-      </>
-    );
-  }
-}
+  return (
+    <>
+      <div id="left">
+        <button type="button" onClick={updateHandleButtonClick}>AC</button>
+        <button type="button" onClick={updateHandleButtonClick}>+/-</button>
+        <button type="button" onClick={updateHandleButtonClick}>%</button>
+        <button type="button" onClick={updateHandleButtonClick}>7</button>
+        <button type="button" onClick={updateHandleButtonClick}>8</button>
+        <button type="button" onClick={updateHandleButtonClick}>9</button>
+        <button type="button" onClick={updateHandleButtonClick}>4</button>
+        <button type="button" onClick={updateHandleButtonClick}>5</button>
+        <button type="button" onClick={updateHandleButtonClick}>6</button>
+        <button type="button" onClick={updateHandleButtonClick}>1</button>
+        <button type="button" onClick={updateHandleButtonClick}>2</button>
+        <button type="button" onClick={updateHandleButtonClick}>3</button>
+        <button id="zero" type="button" onClick={updateHandleButtonClick}>0</button>
+        <button type="button" onClick={updateHandleButtonClick}>.</button>
+      </div>
+      <div id="right">
+        <button type="button" onClick={updateHandleButtonClick}>÷</button>
+        <button type="button" onClick={updateHandleButtonClick}>x</button>
+        <button type="button" onClick={updateHandleButtonClick}>-</button>
+        <button type="button" onClick={updateHandleButtonClick}>+</button>
+        <button type="button" onClick={updateHandleButtonClick}>=</button>
+      </div>
```

Comment:
I like it the way you have created this, and also I think there is a simple approace using destructuring Arguments by storing all the buttons data in an array of objects with their corresponding unique predefind ID's, make it DRY.